.eslintrc.json

```diff
@@ -28,17 +25,41 @@
     {
       "files": ["*.ts", "*.tsx"],
       "extends": ["plugin:@nx/typescript"],
-      "rules": {}
+      "rules": {
+        "@typescript-eslint/no-extra-semi": "error",
+        "no-extra-semi": "off"
+      }
     },
     {
       "files": ["*.js", "*.jsx"],
       "extends": ["plugin:@nx/javascript"],
-      "rules": {}
+      "rules": {
+        "@typescript-eslint/no-extra-semi": "error",
+        "no-extra-semi": "off"
+      }
     },
     {
       "files": "*.json",
       "parser": "jsonc-eslint-parser",
       "rules": {}
+    },
+    {
+      "files": ["*.json"],
```

Comment:
This is the new lint rule which ensures projects have all the dependencies they need.

Reply:
Should this be `*.json` or something more strict like `**/package.json`?

Reply:
This is what I saw in most of the examples. I'm not sure if it maybe considers the package.json too somehow: https://nx.dev/nx-api/eslint-plugin/documents/dependency-checks#manual-setup